app/assets/javascripts/admin/routes/admin-plugins.js.es6

```diff
@@ -4,8 +4,21 @@ export default Ember.Route.extend({
   },
 
   actions: {
-    showSettings() {
-      this.transitionTo('adminSiteSettingsCategory', 'plugins');
+    showSettings(plugin) {
+      const controller = this.controllerFor('adminSiteSettings');
+      this.transitionTo('adminSiteSettingsCategory', 'plugins').then(function() {
```

Comment:
It transitions to this first because if the site settings have never loaded, calling filterContentNow() first will fail - it doesn't have the list.